### ISSUE TITLE
Bug: B-Line GTFS data stale in warehouse since August 18, 2023?

### ISSUE THREAD

**Issue by evansiroky** [cal-itp/data-infra#3176]
**Describe the bug**

When running [this metabase query](https://dashboards.calitp.org/question/2027-b-line-calendar-dates-valid-from-query), it can be seen that the maximum feed valid from date is August 18, 2023. 

Doing a manual spot analysis of [GTFS Schedule](https://github.com/cal-itp/data-infra/files/13622154/google_transit.8.zip) and 
[GTFS-RT data](https://github.com/cal-itp/data-infra/files/13622157/vp-2023-12-08T15-53-53.json) from 2023-12-08, it can be seen that there are clearly some if not most currently running vehicles in the vehicle positions feed that are properly referencing the GTFS Schedule feed.

Therefore, the feed being used in metabase from that time no longer matches the data that is currently being downloaded. It appears that the data in some warehouse tables are stale. This is likely related to a larger issue affecting other parts of the pipeline, such as [Individual Agency GTFS-RT Completeness dashboard](https://dashboards.calitp.org/dashboard/133-individual-agency-gtfs-rt-completeness?text=B-Line%20Schedule).

**To Reproduce**

Run [this metabase query](https://dashboards.calitp.org/question/2027-b-line-calendar-dates-valid-from-query) and check the [Individual Agency GTFS-RT Completeness dashboard](https://dashboards.calitp.org/dashboard/133-individual-agency-gtfs-rt-completeness?text=B-Line%20Schedule).

**Expected behavior**

Schedule data is up-to-date and can properly reference GTFS-RT data.

**Additional context**

none yet.

**Comment by lauriemerrell** [comment]
Investigated this today, investigation notes:
* `feed_valid_from` of August 18 is not necessarily concerning. `feed_valid_from` simply records the first date that a given zipfile version was observed, so we only expect it to update when a changed zipfile is uploaded. [Here's a visualization](https://dashboards.calitp.org/question/2029-feed-valid-from-values-of-current-feeds) of the distribution of `feed_valid_from` for current feeds -- August is not outside the norm.
* The [linked Metabase query showing B-Line feed versions](https://dashboards.calitp.org/question/2027-b-line-calendar-dates-valid-from-query) now shows that a new feed version was created on December 8, 2023 for B-Line.
* I think that what happened is probably that at the time that this was written on Friday, a new zipfile was up on their website but hadn't made its way into the warehouse yet. So the manual checks showed alignment but the warehouse did not have the new zipfile yet. 
* [Looking at total scheduled trips by day](https://dashboards.calitp.org/question/2028-b-line-scheduled-service-last-3-months), it looks like there was reasonable continuity for B-Line in terms of scheduled service through November to the present.
* However, this does not explain why the RT completeness dropped to 0 on November 19. Looking into that, I found that all the trip IDs present on November 19 in the RT data did not match the IDs present in the scheduled data for the same date. So both the schedule and the RT data were downloaded successfully but the IDs present for each did not align. I think that basically their RT feed switched to using their updated feed a few weeks before that updated feed was publicly available. See [this Metabase question](https://dashboards.calitp.org/question/2030-b-line-november-19-2023-trip-id-investigation) which shows that the trip IDs present in the RT data on November 19 did not appear in the scheduled data until December 10. 

TLDR: I do not think that there is any pipeline or technical issue here, believe that the issue was with the agency data. Request to close.

**Comment by evansiroky** [comment]
Thanks for investigating more thoroughly. I'm closing as I do now believe this is not a bug.
